bridge/dd_wrap_autoloader.php

```diff
@@ -12,16 +12,25 @@
 
 $dd_autoload_called = false;
```

Comment:
Since these variables get dumped into the global scope even with the namespacing, can we wrap everything from this line down with a self-calling function?

```php
call_user_func(function () {
    $dd_autoload_called = false;
    // ....
});
```

That will prevent these variables from showing up in user's apps to potentially cause unexpected behavior. :)

<img width="375" alt="screen shot 2019-01-25 at 12 53 18 pm" src="https://user-images.githubusercontent.com/578780/51763846-7538f000-20a1-11e9-8045-8623b50d8764.png">